src/App.tsx

```diff
@@ -1,11 +1,84 @@
-import React from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
 
 import './App.scss';
+import { useState, useEffect } from 'react';
+import {
+  Routes,
+  Route,
+  NavLink,
+  Navigate,
+  useLocation,
+} from 'react-router-dom';
+import classNames from 'classnames';
+import { getPeople } from './api';
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { Person } from './types/Person';
+import { NotFoundPage } from './components/NotFoundPage';
+
+library.add(fas);
+
+type Status = { isActive: boolean };
+
+const getActiveClasses = (status: Status) => classNames(
+  { 'link--selected': status.isActive },
 );
 
-export default App;
+export const App = () => {
+  const location = useLocation();
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(peopleFromServer => {
+        const preparedPeople = peopleFromServer.map(p => ({ ...p }));
+
+        preparedPeople.forEach(person => {
+          Object.assign(person, {
+            mother: preparedPeople.find(m => m.name === person.motherName)
+            || null,
+            father: preparedPeople.find(f => f.name === person.fatherName)
+            || null,
+          });
+        });
```

Comment:
why do you iterate through array 2 times, your first `map` call does nothing, move all this logic to your `map` and avoid using `forEach`